src/intro/tour.rst

```diff
@@ -58,7 +63,12 @@ All we added to the previous request is the _all_dbs string.
 
 The response should look like::
 
-    ["_replicator","_users"]
+    ["_global_changes","_replicator","_users"]
+
+.. note::
+    In case this returns an empty Array for you, it means you haven't finished
```

Comment:
Should "Array" be capitalized?  I would think no, it doesn't seem like a proper noun to me, but I may be wrong in this context?